### PR TITLE
Edit Popup to show multiple handler options to choose from.

### PR DESCRIPTION
**Pull request by ustundag**
multiple-handlers

### CODE REVIEW

**Comment by ustundag** [comment]
I've combined popup processes in PopupService and added missing handler choice as an option in the popup, like Navigate to an existing handler or Create a new handler

P.S.: There is a 1 todo that still remains a challenge for me. I'd appreciate it if you propose a more safe way.

File: src/main/kotlin/org/bilalkilic/kediatrhelper/listeners/KediatrHandlerClassMarker.kt [hunk 1]
```diff
@@ -63,45 +60,75 @@ class KediatrHandlerClassMarker : LineMarkerProvider {
...
+                options.add(PopupItem(HandlerType.BASIC, POPUP_STEP_HANDLER_BASIC, mainClass, ItemScope.CREATE))
+                options.add(PopupItem(HandlerType.ASYNC, POPUP_STEP_HANDLER_ASYNC, mainClass, ItemScope.CREATE))
             }
-
-            override fun onChosen(selectedValue: PopupStepForHandler, finalChoice: Boolean): PopupStep<*>? {
-                mainClass.project.service<CreateNewHandlerService>().create(mainClass, selectedValue)
-                return PopupStep.FINAL_CHOICE
+            handlers.size == 1 -> {
+                val handler = handlers.firstOrNull() ?: return
+                // todo : not sure that it is a good way of converting PsiClass to KtClass ?
```
Comment (bilal-kilic) [inline src/main/kotlin/org/bilalkilic/kediatrhelper/listeners/KediatrHandlerClassMarker.kt:80]: `KtPsiClassWrapper(it).psiOrParent as KtClass `
It not much better but I've managed to do it like this. We're literally blind working with kotlin plugin 😢

**Review by bilal-kilic — APPROVED** [review]

File: src/main/kotlin/org/bilalkilic/kediatrhelper/listeners/KediatrHandlerClassMarker.kt (hunk 1 above)
Comment (ustundag) [reply src/main/kotlin/org/bilalkilic/kediatrhelper/listeners/KediatrHandlerClassMarker.kt:80]: thanks for the suggestion but somehow it didnt work when navigating to existing handlers. no exception nor warning. weird :/

Comment (ustundag) [reply src/main/kotlin/org/bilalkilic/kediatrhelper/listeners/KediatrHandlerClassMarker.kt:80]: did you manage to get it working by using this wrapper?

Comment (bilal-kilic) [reply src/main/kotlin/org/bilalkilic/kediatrhelper/listeners/KediatrHandlerClassMarker.kt:80]: Nevermind it doesn't work for me either.

**Pull request merged**